src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -535,19 +535,19 @@ void EnGinkoMan_SetupBankAward2(EnGinkoMan* this) {
     this->actionFunc = EnGinkoMan_BankAward2;
 }
 
-// separate function to handle bank rewards... if the bank has a parent actor? might be unused
+// separate function to handle bank rewards... called while the player is receiving the award
```

Comment:
I think we should use "reward" consistently... the question is whether to do that for the function names too. Also looks like the first one is more `StartGiveReward` and the second `GiveReward`. This is not a terribly uncommon pattern.

Reply:
I talked about this in discord, but my understanding of `EnGinkoMan_BankAward2` is that it runs after the player receives an item. It's not responsible for giving an item, but rather handles the logic after the player has been given an item.
```suggestion
// Called after the player has obtained the reward
```

Reply:
The function is called repeatedly while the player is holding the item, reading the item text etc. Additional checks within the function are responsible for checking if the player has closed the item message box.